src/test/groovy/com/synacy/gradprogram/spock/exercise/OrderingServiceSpec.groovy

```diff
@@ -0,0 +1,187 @@
+        given:
+        List<Item> testListItem = [food, gadget, appliance, clothing]
+        def testCart = new Cart(UUID.randomUUID(), testListItem)
+
+        when:
+        double testTotal = orderingService.calculateTotalCostOfCart(testCart)
+
+        then:
+        55d == testTotal
+    }
+
+    def "isCartEligibleForDiscount should respond with true if total cart price is greater than 50.0 and size is greater than 5"() {
+
+        given:
+        List<Item> testListItem = [food, gadget, appliance, clothing, food2, gadget2]
+        def testCart = new Cart(UUID.randomUUID(), testListItem)
+        boolean isExpectedEligible = true
+
+        when:
+        boolean isEligible = orderingService.isCartEligibleForDiscount(testCart)
+
+        then:
+        isExpectedEligible == isEligible
+    }
+
+    def "isCartEligibleForDiscount should respond with false if cart price is less than 50.0 and cart size is less than 5"() {
+
+        given:
+        List<Item> testListItem = [food, gadget, clothing]
+        def testCart = new Cart(UUID.randomUUID(), testListItem)
+        boolean isNotEligible = false
+
+        when:
+        boolean isEligible = orderingService.isCartEligibleForDiscount(testCart)
+
+        then:
+        isNotEligible == isEligible
+    }
+
+    def "isCartEligibleForDiscount should respond with false if cart price is greater than 50.0 but cart size is less than 5"() {
+
+        given:
+        List<Item> testListItem = [food, gadget, appliance, clothing]
+        def testCart = new Cart(UUID.randomUUID(), testListItem)
+        boolean isNotEligible = false
+
+        when:
+        boolean isEligible = orderingService.isCartEligibleForDiscount(testCart)
+
+        then:
+        isNotEligible == isEligible
+    }
+
+    def "isCartEligibleForDiscount should respond with false if cart price is less than 50.0 but cart size is greater than 5"() {
+
+        given:
+        List<Item> testListItem = [food2, gadget2, clothing2, gadget, clothing, appliance2]
+        def testCart = new Cart(UUID.randomUUID(), testListItem)
+        boolean isNotEligible = false
+
+        when:
+        boolean isEligible = orderingService.isCartEligibleForDiscount(testCart)
+
+        then:
+        isNotEligible == isEligible
+    }
+
+    def "ApplyDiscountToCartItems should respond with discounted cost"() {
+        given:
+        List<Item> testListItem = [food, gadget, appliance, clothing, food2, gadget2]
+
+        def testCart = new Cart(UUID.randomUUID(), testListItem)
+
+        when:
+        orderingService.applyDiscountToCartItems(testCart)
+
+        then:
+        1.5d == testListItem[0].getCost()
+        1d == testListItem[1].getCost()
+        2d == testListItem[2].getCost()
+        1d == testListItem[3].getCost()
+        0.5d == testListItem[4].getCost()
+        0.5d == testListItem[5].getCost()
+    }
+
+    def "ApplyDiscountToCartItems should not respond with discounted cost"() {
```

Comment:
Having something like "should not do anything" or "should not apply discount to the prices" is better here.